src/test/java/subway/Line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,258 @@
+package subway.Line;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.aspectj.lang.annotation.Before;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.Sql.ExecutionPhase;
+
+@SuppressWarnings("NonAsciiCharacters")
+@DisplayName("지하철 노선 관련 기능")
+@Sql(scripts = {"/truncate.sql"}, executionPhase = ExecutionPhase.BEFORE_TEST_METHOD)
+@SpringBootTest(webEnvironment = WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest extends LineAcceptanceFixture {
+    Long 신사역_ID;
+    Long 강남역_ID;
+    Long 청량리_ID;
+    Long 서울숲_ID;
+
+    @BeforeEach
+    void setUp() {
+        신사역_ID = requestCreateStation("신사역")
+                .jsonPath()
+                .getObject("id", Long.class);
+
+        강남역_ID = requestCreateStation("강남역")
+                .jsonPath()
+                .getObject("id", Long.class);
+
+        청량리_ID = requestCreateStation("청량리")
+                .jsonPath()
+                .getObject("id", Long.class);
+        서울숲_ID = requestCreateStation("서울숲")
+                .jsonPath()
+                .getObject("id", Long.class);
```

Comment:
```suggestion
        신사역 = requestCreateStation("신사역")
                .jsonPath()
                .getObject("id", Long.class);

        강남역 = requestCreateStation("강남역")
                .jsonPath()
                .getObject("id", Long.class);

        청량리 = requestCreateStation("청량리")
                .jsonPath()
                .getObject("id", Long.class);
        서울숲 = requestCreateStation("서울숲")
                .jsonPath()
                .getObject("id", Long.class);
```

`_ID`는 제거해줘도 좋을 것 같네요~